wiki_migrate.py

```diff
@@ -52,6 +68,83 @@ def main():
         os.chdir(start_dir)
 
 
+def _mirate_pq_dump(db_file, target_repo):
+    """
+    Generate files based on pg_dump file.
+
+    pg_dump  --no-owner --data-only  --file=trac-wiki.dump --table=wiki trac
+    """
+
+    copy_started = False
+    # The dump is not sorted by timestamp, so we need to manually sort it
+    # and keep all pages in memory.
+    changes = []
+
+    with open(db_file, 'rb') as stream:
+        while True:
+            line = stream.readline()
+
+            if not line:
+                # End of stream.
+                # Most likely this is not heat.
+                break
+
+            if line == b'\\.\n':
+                # End of COPY dump.
+                break
+
+            if line.startswith(b'COPY '):
+                # We can start to process the next line.
+                copy_started = True
```

Comment:
If this works, then it's fine! But I couldn't try it out or see the output, because I don't have a dump.